cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -201,7 +200,7 @@ private boolean saveAccessToken(AccessToken accessToken, String hostname) {
     File accessTokenFile = getAccessTokenFile(hostname);
 
     try {
-      if (accessTokenFile.createNewFile()) {
+      if (accessTokenFile.exists() || accessTokenFile.createNewFile()) {
```

Comment:
The reason for the condition is to allow only these flows:
(a) file exists -> overwrite
(b) file doesn't exist -> create new file -> write

but disallow
(c) file doesn't exist -> failed to create new file -> write

Instead of throwing an exception for (c), I allow CLI to fail silently since saving access token isn't critical.

Though I suppose we can simply do

```
if (!accessTokenFile.createNewFile()) {
  return false;
}
Files.write(accessToken.getValue(), accessTokenFile, Charsets.UTF_8);
return true;
```
which is probably more clear.